examples/quickstart-catboost/quickstart_catboost/client_app.py

```diff
@@ -0,0 +1,64 @@
+"""quickstart-catboost: A Flower / CatBoost app."""
+
+from catboost import CatBoostClassifier, Pool
+import json
+from quickstart_catboost.task import (
+    convert_to_catboost,
+    convert_to_model_dict,
+    load_data,
+)
+
+from flwr.client import ClientApp
+from flwr.common import Context, Message, ConfigRecord, RecordDict
+
+# Flower ClientApp
+app = ClientApp()
+
+
+@app.train()
+def train(msg: Message, context: Context) -> Message:
+
+    # Load partition
+    partition_id = context.node_config["partition-id"]
+    num_partitions = context.node_config["num-partitions"]
+    (X_train, y_train), (X_test, y_test), cat_features = load_data(
+        partition_id, num_partitions
+    )
+
+    # Instantiate local model
+    iterations = context.run_config["local-epochs"]
+    learning_rate = context.run_config["learning-rate"]
+    depth = context.run_config["depth"]
+    cbc = CatBoostClassifier(
+        iterations=iterations,
+        learning_rate=learning_rate,
+        depth=depth,
+        random_seed=42,
+        verbose=0,
+        cat_features=cat_features,
+    )
+
+    # Load global model
+    global_model_dict = msg.content["gmodel"]["model"]
+    cbc_init = convert_to_catboost(global_model_dict) if global_model_dict else None
+
+    # Local training
+    cbc.fit(X_train, y_train, init_model=cbc_init)
+
+    # Evaluation
+    eval_pool = Pool(data=X_test, label=y_test, cat_features=cat_features)
+    metrics = cbc.eval_metrics(eval_pool, metrics=["AUC"])
+    auc = metrics["AUC"][-1]
+
+    # Extract boosted trees
+    model_dict = convert_to_model_dict(cbc)
+    num_trees = len(model_dict["oblivious_trees"])
+    model_dict["oblivious_trees"] = model_dict["oblivious_trees"][
+        num_trees - iterations : num_trees
+    ]
```

Comment:
Could this be done before serializing the model? I'd say is better to add a comment explaining what these lines do